code/modules/atmospherics/machinery/portable/portable_atmospherics.dm

```diff
@@ -141,7 +141,7 @@
 				return
 			else
 				var/obj/machinery/atmospherics/components/unary/portables_connector/possible_port = locate(/obj/machinery/atmospherics/components/unary/portables_connector) in loc
-				if(!possible_port)
+				if(!possible_port || bounds_dist(src, possible_port) >= -5) // canister needs to be at least 5 pixels overlapping the port

```

Comment:
you can just do `locate() in bounds(src, -5)`

At least I think negative numbers works with that proc

Reply:
That’s only offset in the x, I tried that first you could do bounds(src, -5, -5) but that’d also still be off because then you’d need to be 5 pixels overlapping in the X AND Y